ocaml/xapi/repository.ml

```diff
@@ -228,14 +229,15 @@ let sync ~__context ~self ~token ~token_id ~username ~password =
     in
     Xapi_stdext_pervasives.Pervasiveext.finally
       (fun () ->
-        let config_repo params =
-          let Pkg_mgr.{cmd; params} =
-            "--save" :: params |> fun config ->
-            Pkgs.config_repo ~repo_name ~config
-          in
-          ignore (Helpers.call_script ~log_output:Helpers.On_failure cmd params)
+        let config_repo config =
+          if List.length config <> 0 then (* Set params to yum/dnf *)
```

Comment:
It would be better to test against `[]`.

Reply:
xenrt has covered that.